third-party/py/twitter-commons/src/python/BUCK

```diff
@@ -2,9 +2,6 @@ python_library(
   name = 'twitter-commons',
   base_module = '',
   srcs = glob(['**/*.py']),
-  deps = [
-    '//third-party/py/setuptools:pkg_resources',
-  ],
```

Comment:
Why remove this dep?  I think the pex sources import `pkg_resources`, which means there should be a dep here.